middleware/TelemetryHandler.md

```diff
@@ -22,7 +22,11 @@ Provide a mandatory middleware component that attaches metadata to a Graph reque
     ```
     SdkVersion: MyApp/1.0, graph-dotnet/1.15.0, graph-dotnet-core/1.16.0 (featureUsage=0f)
     ```
-    
+
+- The Telemetry Handler MUST NOT append values to an existing SdkVersion header in the case of an retry scenario. The handler can check for the existance of the `retry-attempt` header to determine whether the request is a retry.
+
+- The Telemetry Handler MUST be the final custom handler so that it can aggregate information about handlers earlier in the chain.
```

Comment:
In the case of typescript (and Go if I remember right), the last middleware is actually the one that makes the request. I understand this is language specific, but could we rephrase that to be inclusive of that technical constraint?

Reply:
Would it be also okay to place the `Telemetry Handler` before the retry handler? 
This way when the retry occurs, it would not go through the `Telemetry Handler` again and avoid the scenario.

Reply:
I don't think we should assume any position in the internal behaviour of the handler since users can re shuffle them

Reply:
Agreed that we shouldn't assume position but we can provide guidance on where it should be. Will change MUST to should and clarify about it not necessarily being the final handler.